src-ghc/Pact/Bench.hs

```diff
@@ -384,4 +399,27 @@ main = do
       , benchNFIO "round4" $ runPactExec def "round4" [] Null Nothing pureDb round4
       ]
     , benchNFIO "time" $ fmap fst <$> evalReplEval def replS (mapM eval timeTest)
+    , bgroup "defun"
+      [ bgroup "return-type-tc"
```

Comment:
Fix the name here to be more descriptive of what we're actually benchmarking (effect of size of arguments).